### ISSUE TITLE
Can this plugin run code in docker container and  get the result in quickfix list?

### ISSUE THREAD

**Issue by meicale** [skywind3000/asyncrun.vim#273]
It is a cool plugin and I like it. I use docker container as the development environment as well, which is easy to share and manage. How can I run the code in this docker container and can consume put the result in the quick-fix list as well. Maybe, I can be done by substitute the command to command with prefix “docker exec [args]". Where can I change the command actually run by this plugin? The other thing is how to get the result generated in docker container can copy it back to the host and set it in quick-fix list. 
Thank you in advance!

**Comment by skywind3000** [comment]
see the command modifier: https://github.com/skywind3000/asyncrun.vim#command-modifier

you can define a new modifier in `~/.vim/autoload/asyncrun/program` or any sub-directory named `autoload/asyncrun/program` inside a runtime-path.

here is an example: https://github.com/skywind3000/asyncrun.vim/blob/master/autoload/asyncrun/program/shebang.vim

this script allows you use:

    :AsyncRun -program=shebang  ls -la

the command `ls -la` will be processed and replaced by `shebang.vim` before execution.

**Comment by meicale** [comment]
I don't think it is that straightforward. I use a compiler to deal with my python trackback and I have to use the same file directory between host and container. Can this be solved to provide something like path_mapping in neovim dap debugger remote? We can `sed` the result and replace with the right file path of the trace back result. I think it is quite common helpful if we can support running code in docker easily. The solution to this is just a path mapping and helper function. I know little about the vim script and can this done using lua?
